apps/web/src/pages/integrations/IntegrationsStoreModal.tsx

```diff
@@ -21,6 +22,7 @@ import { Close } from '../../design-system/icons/actions/Close';
 import { useProviders } from './useProviders';
 import { useSegment } from '../../components/providers/SegmentProvider';
 import { IntegrationsStoreModalAnalytics } from './constants';
+import { NovuSMSProviderModal } from './components/NovuSMSProviderModal';
```

Comment:
```suggestion
import { NovuSmsProviderModal } from './components/NovuSmsProviderModal';
```
So we follow the convention we tend to use (like in `SmsProviderIdEnum`.